lib/metrics/bootstrapTime.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+const Gauge = require('../gauge');
+
+let perf_hooks;
+
+try {
+	// eslint-disable-next-line
```

Comment:
can you disable the specific rule?

Reply:
fixed

Reply:
I'm sorry, but its looks like I can not disable only specific rule.
For node 6.x I should use 
`//eslint-disable-next-line node/no-missing-require`
But for newer versions of node I should use  
`//eslint-disable-next-line node/no-unsupported-features/node-builtins`
But I can not specify both rules. So I rolled back this to just 
`//eslint-disable-next-line `